LLama/Native/NativeApi.Load.cs

```diff
@@ -80,47 +87,7 @@ private static void SetDllImportResolver()
 
         private static void Log(string message, LLamaLogLevel level)
         {
-            if (!enableLogging)
-                return;
-
-            if ((int)level > (int)logLevel)
-                return;
-
-            var fg = Console.ForegroundColor;
-            var bg = Console.BackgroundColor;
-            try
-            {
-                ConsoleColor color;
-                string levelPrefix;
-                if (level == LLamaLogLevel.Debug)
-                {
-                    color = ConsoleColor.Cyan;
-                    levelPrefix = "[Debug]";
-                }
-                else if (level == LLamaLogLevel.Info)
-                {
-                    color = ConsoleColor.Green;
-                    levelPrefix = "[Info]";
-                }
-                else if (level == LLamaLogLevel.Error)
-                {
-                    color = ConsoleColor.Red;
-                    levelPrefix = "[Error]";
-                }
-                else
-                {
-                    color = ConsoleColor.Yellow;
-                    levelPrefix = "[UNK]";
-                }
-
-                Console.ForegroundColor = color;
-                Console.WriteLine($"{loggingPrefix} {levelPrefix} {message}");
-            }
-            finally
-            {
-                Console.ForegroundColor = fg;
-                Console.BackgroundColor = bg;
-            }
+            NativeLibraryConfig.Instance.LogCallback?.Invoke(level, message);
```

Comment:
Since log callback now used by both llama.cpp and LLamaSharp, and logger-base callback implementation relies on trailing `\n`, we should probably add trailing `\n` here to the message.

Reply:
Done.